### DISCUSSION TITLE
What  is the max I can downgrade on kernver 4

### DISCUSSION THREAD

**Discussion by OPcoolman** [3kh0/ext-remover#1319] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I wanted to know this because I wanted to downgrade

**Comment by schoolexploitkid** [comment, marked as answer]
v125

**Comment by ghost** [comment]
How would you downgrade for kernver 4 cuz olybmmer doesn´t work

**Reply by OPcoolman** [reply]
It’s a different method I think

**Reply by ghost** [reply]
Do you know where I can find it?

**Reply by OPcoolman** [reply]
Also how do I downgrade to 125 and what method

**Reply by prjamming** [reply]
Download the bin.zip file for your board from chrome100.dev Make sure to download the recovery image with v125, not an rma shim. Then use a personal laptop with the chromebook recovery extension installed. Click the cogwheel in the top right of the recovery extension and click use custom image. Select the bin.zip file that you downloaded and click continue. You need a usb drive w/ atleast 8gb. After you've made your recovery usb, powerwash with ESC + ↻ + ⏻, and plug in your recovery usb. It WILL wipe all of your data so back up everything to google drive. Then have fun. I recommend doing rigtools but it is up to you.

**Reply by schoolexploitkid** [reply]
u cant use olybmmer
